app/locales/fr.json

```diff
@@ -1,87 +1,87 @@
 {
 	"commonText": {
 		"appName": "Forest Watcher 2.0",
-		"next": "Next",
-		"back": "Back",
-		"finish": "Finish",
-		"setUp": "Set up",
-    "cancel": "Cancel",
-    "confirm": "Confirm"
+		"next": "Suivant",
+		"back": "Retourner",
+		"finish": "Terminer",
+		"setUp": "Installer",
+    "cancel": "Annuler",
+    "confirm": "Confirmer"
 	},
 	"dashboard": {
-		"title": "Dashboard",
-		"alerts": "See my alerts",
-		"reports": "Make a report",
-		"myReports": "My reports",
-		"uploadData": "Saved and uploaded data",
-		"map": "Map"
+		"title": "Panel",
+		"alerts": "Voir mes alertes",
+		"reports": "Faire un rapport",
+		"myReports": "Mes rapports",
+		"uploadData": "Données sauvegardées et téléchargées",
+		"map": "Carte"
 	},
 	"login": {
-		"title": "Login",
-		"introductionText": "Sign in with a MyGFW account",
+		"title": "S'identifier",
+		"introductionText": "Connectez-vous avec un compte MyGFW",
 		"facebookTitle": "Facebook",
 		"twitterTitle": "Twitter",
 		"googleTitle": "Google",
-		"brazilTitle": "Brazil",
-		"colombiaTitle": "Colombia",
-		"indonesiaTitle": "Indonesia"
+		"brazilTitle": "Brésil",
+		"colombiaTitle": "Colombie",
+		"indonesiaTitle": "Indonésie"
 	},
 	"countries": {
-		"searchPlaceholder": "Search for a country"
+		"searchPlaceholder": "Rechercher un pays"
 	},
 	"alerts": {
-		"title:": "Alerts",
-		"seeAll": "See all",
-		"alert": "alert"
+		"title:": "Alertes",
+		"seeAll": "Voir tout",
+		"alert": "alerte"
 	},
 	"boundaries": {
-		"chooseBoundaries": "Now, choose the boundaries",
-		"selectProtectedArea": "Select a protected area",
-		"drawArea": "Draw an area"
+		"chooseBoundaries": "Maintenant, choisissez les limites",
+		"selectProtectedArea": "Sélectionnez une zone protégée",
+		"drawArea": "Dessinez une zone"
 	},
 	"setupCountry": {
-		"welcomeText": "Hi",
-		"secondWelcomeText": "please set up an area",
-		"firstInstruction": "First, select your country of interest"
+		"welcomeText": "Salut",
+		"secondWelcomeText": "veuillez créer une zone",
+		"firstInstruction": "Tout d'abord, sélectionnez votre pays d'intérêt"
 	},
 	"setupDrawAreas": {
-		"tapInstruction": "Tap on map to draw"
+		"tapInstruction": "Appuyez sur la carte pour dessiner"
 	},
 	"setupHeader": {
-		"errorFirst": "Invalid prop",
-		"errorSecond": "supplied to",
-		"errorThird": "It is required if you show back button"
+		"errorFirst": "Prop non valide",
+		"errorSecond": "fourni à",
+		"errorThird": "Il est nécessaire si vous affichez le bouton"
```

Comment:
Requis pour afficher le button de retour

I don't understand the English sentence but the current French version sounds weird.